packages/shared/src/store/sync.ts

```diff
@@ -1425,145 +1434,139 @@ export const syncStart = async (alreadySubscribed?: boolean) => {
-      logger.crumb(`finished channel predictive sync`);
-    }),
-    syncSystemContacts({ priority: SyncPriority.Low }).then(() => {
-      logger.crumb(`finished syncing system contacts`);
-    }),
-  ];
-
-  await Promise.all(lowPriorityPromises)
-    .then(() => {
-      logger.crumb(`finished low priority sync`);
-    })
-    .catch((e) => {
-      logger.trackError(AnalyticsEvent.ErrorSyncStartLowPriority, {
-        errorMessage: e.message,
+    } catch (err) {
+      logger.trackError(AnalyticsEvent.ErrorSyncStartHighPriority, {
+        errorMessage: err.message,
       });
-    });
-
-  // post sync initialization work
-  await verifyUserInviteLink();
+    }
 
-  const hostingUser = await db.hostingUserId.getValue();
-  await api.getHostingUser(hostingUser);
+    const lowPriorityPromises = [
+      alreadySubscribed
+        ? Promise.resolve()
+        : setupLowPrioritySubscriptions({
+            priority: SyncPriority.Medium,
+          }).then(() => logger.crumb('subscribed low priority')),
+      resetActivity({ priority: SyncPriority.Medium + 1, retry: true }).then(
+        () => logger.crumb(`finished resetting activity`)
+      ),
+      // if we had cached contacts, we refresh them here with low priority
+      didLoadCachedContacts
+        ? syncContacts({ priority: SyncPriority.Medium + 1, retry: true }).then(
+            () => logger.crumb(`finished syncing contacts`)
+          )
+        : Promise.resolve(),
+      syncSettings({ priority: SyncPriority.Medium }).then(() =>
+        logger.crumb(`finished syncing settings`)
+      ),
+      syncVolumeSettings({ priority: SyncPriority.Low }).then(() =>
+        logger.crumb(`finished syncing volume settings`)
+      ),
+      syncStorageSettings({ priority: SyncPriority.Low }).then(() =>
+        logger.crumb(`finished initializing storage`)
+      ),
+      syncPushNotificationsSetting({ priority: SyncPriority.Low }).then(() =>
+        logger.crumb(`finished syncing push notifications setting`)
+      ),
+      syncAppInfo({ priority: SyncPriority.Low }).then(() => {
+        logger.crumb(`finished syncing app info`);
+      }),
+      syncRelevantChannelPosts({ priority: SyncPriority.Low }).then(() => {
+        logger.crumb(`finished channel predictive sync`);
+      }),
+      syncSystemContacts({ priority: SyncPriority.Low }).then(() => {
+        logger.crumb(`finished syncing system contacts`);
+      }),
+    ];
+
+    await Promise.all(lowPriorityPromises)
+      .then(() => {
+        logger.crumb(`finished low priority sync`);
+      })
+      .catch((e) => {
+        logger.trackError(AnalyticsEvent.ErrorSyncStartLowPriority, {
+          errorMessage: e.message,
+        });
+      });
 
-  const ship = await db.hostedUserNodeId.getValue();
-  if (ship) {
-    await api.getShip(ship);
+    // post sync initialization work
+    await verifyUserInviteLink();
+    db.userHasCompletedFirstSync.setValue(true);
+  } finally {
+    isSyncing = false;
   }
```

Comment:
did we mean to delete these?

Reply:
yeah, I don't think they were actually needed